pkg/features/features.go

```diff
@@ -26,6 +26,10 @@ const (
 	// alpha: v0.4
 	// beta: v1.2
 	Topology featuregate.Feature = "Topology"
+
+	// owner: @pohly
+	// alpha: 2.0.0
+	CentralCapacity featuregate.Feature = "CentralCapacity"
```

Comment:
In the KEP, I had described separate command line options for enabling the different storage capacity tracking modes. Conceptually those were similar (and inspired) by the feature mechanism in Kubernetes, with a single command line option taking different strings that form a set.

While working on the implementation, I found that external-provisioner is already using the Kubernetes feature infrastructure. I'd like to propose that we just use that also for the new feature capacity tracking. Each mode of producing CSIStorageCapacity then maps to a feature. Sp this is now the new option:
```
--feature-gates CentralCapacity=on
```

@jsafrane: what do you think?

Reply:
Feature gates is for code that's graduating and they are eventually removed. If I understand the proposal correctly, we  still need a parameter that tells which capacity mode to use when the feature is GA and we remove its feature gate (if we have one).

IMO, we don't really need feature gate on this, we may add `-enable-capacity=central` and mark it as alpha.

Reply:
Okay. So should I use a simple boolean `-enable-capacity-central` or `-enable-capacity=central`? I have the code for the latter in https://github.com/kubernetes-csi/external-provisioner/pull/450/files#diff-513b7bafc46ec7994a7f0bc31fd799cc

Reply:
I like the `-enable-capacity=central`, as described in the KEP.

Reply:
Done.